src/pages/projectSettings/tabs/GeneralTab/getFormSchema.tsx

```diff
@@ -147,20 +148,18 @@ export const getFormSchema = (
             title: "Default Logger",
             properties: {
               defaultLogger: {
-                default: null,
-                type: [
-                  "string",
-                  ...insertIf(
-                    projectType === ProjectType.AttachedProject,
-                    "null"
-                  ),
-                ],
-                enum: [
-                  ...validDefaultLoggers,
-                  ...insertIf(
-                    projectType === ProjectType.AttachedProject,
-                    null
-                  ),
+                type: "string" as "string",
+                oneOf: [
```

Comment:
The `oneOf` and `enum` patterns are different ways to do the same thing; use `oneOf` to align more with implementation elsewhere